src/main-page/navbar/main navbar/navbar.js

```diff
@@ -3,7 +3,7 @@ import { NavLink } from "react-router-dom";
 import { connect } from "react-redux";
 import "./navbar.css";
 const Navbar = (props) => {
-  const click = 0;
+  let click = 0;
   const myFunction = () => {
```

Comment:
Can we name this component `Navbar`

Reply:
Also, I see a space in path can you fix that to use recommended naming strategy.

Reply:
@aashutoshrathi  sir can you please elaborate it a bit.

Reply:
- `src/main-page/navbar/main navbar/navbar.js` the directory here contains space in name.
- Also, the component's name is not correct here, please change that too